client/curriculum/html/LinksAndButtons.tsx

```diff
@@ -0,0 +1,882 @@
+            alt="Link-uri către numere de telefon"
+            demo="/demo/html/linkuri-catre-numere-de-telefon"
+          />
+          <p>
+            Împreuna cu atributul
+            {' '}
+            <strong className="formatted">download</strong>
+            , putem instrui
+            browser-ul să downloadeze fișierul din cadrul link-ului în loc să îl
+            deschidă în pagina curentă:
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="/files/test.pdf" download>Download PDF</a>
+`}
+          />
+          <LessonFigure
+            withBorder
+            src="/images/lessons/links-and-buttons/download.png"
+            alt="Atributul download"
+            demo="/demo/html/linkuri-atributul-download"
+          />
+          <p>
+            Oky, acum că ne-am familiarizat cu situații din practică unde putem
+            folosi link-uri, ne-am obișnuit puțin cu markup-ul și cu câteva
+            dintre atributele ce pot însoți elementul, putem să trecem la ceva
+            detalii legate de cele mai bune practici pe care să le avem în
+            vedere.
+          </p>
+        </section>
+        <section>
+          <LessonHeading as="h3" id="atributul-target-blank">
+            Atributul target=”_blank”
+          </LessonHeading>
+          <p>
+            În afară de atributele prezentate, un alt atribut folosit este
+            {' '}
+            <strong className="formatted"> target</strong>
+            {' '}
+            , care ne indică unde
+            să deschidem link-ul din href.
+          </p>
+          <p>
+            Ca și developeri cel mai des ne vom întâlni cu situația de a folosi
+            {' '}
+            <strong>
+              target=”_blank” (link-ul va fi deschis într-o fereastră sau un tab
+              nou)
+            </strong>
+          </p>
+        </section>
+        <section>
+          <LessonHeading as="h3" id="bune-practici">
+            Bune practici
+          </LessonHeading>
+          <p>
+            Când folosim valoarea _blank, e recomandat să adăugăm un nou atribut
+            pentru securitate și anume :
+            {' '}
+            <strong className="formatted">
+              rel cu valorile ”noopener noreferrer”.
+            </strong>
+            {' '}
+            Atributul
+            {' '}
+            <strong className="formatted">rel</strong>
+            {' '}
+            poate avea mai
+            multe valori, dar le vom prezenta aici doar pe cele 2 recomandate a
+            se folosi împreuna cu atributul target=”_blank”.
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="http://frontend.ro" target="_blank" rel="noopener noreferrer">
+  Link-ul se deschide într-un nou tab
+</a>
+            `}
+          />
+          <LessonFigure
```

Comment:
Remove demo completely, similar with previous ones. Plus the difference is actually from a security point.